src/main/resources/blockchain/default-genesis.json

```diff
@@ -1,5 +1,5 @@
 {
-  "difficulty": "0x0400000000",
+  "difficulty": "0x040000",
```

Comment:
Was this intentional? I'm somewhat confused because according to YP neither value is correct. It should be 2^17 which is `0x20000`